src/deezer/client.py

```diff
@@ -352,6 +352,59 @@ def get_user(self, user_id: int | None = None) -> User:
         user_id_str = str(user_id) if user_id else "me"
         return self.request("GET", f"user/{user_id_str}")
 
+    def get_user_recommendations_track(self, **kwargs) -> PaginatedList[Track]:
```

Comment:
Renaming the functions is definitely a good idea. I will do that right away! Originally i actually put it in the `User` resource but i moved it to `Client` because the function only works on the `/me/` user. But if you want, we can put it back in the `User` resource.

Reply:
I see, you're right. Let's stick with the ones you've defined